packages/cli/src/preview-server/routes.ts

```diff
@@ -148,9 +148,10 @@ const handleOpenInEditor = async (
 		const stack = body.stack as SymbolicatedStackFrame;
 
 		const guess = await editorGuess;
+		const defaultCodeEditor = { process: 'code', command: 'code' }
 		const didOpen = await launchEditor({
 			colNumber: stack.originalColumnNumber as number,
-			editor: guess[0],
```

Comment:
Let's not default to VS Code, instead if the editor was not guessed, let's not display the menu at all.